### ISSUE TITLE
nvda's autorun does not works

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2216]
**Reported by ateu on 2012-04-04 22:13**
I tested with xp and windows 7.
The autorun is created, but nvda is not automaticaly started.

**Comment by jcsteh** [comment]
NVDA no longer provides the option to creat autorun.inf (61c2d1c2), as it doesn't work at all for USB drives now. Even on optical media, it still requires manual intervention from the user when the dialog appears.
